front/static/front/js/decks.js

```diff
@@ -1,3 +1,4 @@
+url_home="http://127.0.0.1:8000"
```

Comment:
Why did you hard code local url?

Reply:
because I was stupid and ... I forget delete this line before